pilot/pkg/config/aggregate/config.go

```diff
@@ -110,13 +111,12 @@ func (cr *store) List(typ config.GroupVersionKind, namespace string) []config.Co
 	}
 	var configs []config.Config
 	// Used to remove duplicated config
-	configMap := sets.New[string]()
+	configMap := sets.New[types.NamespacedName]()
 
 	for _, store := range cr.stores[typ] {
 		storeConfigs := store.List(typ, namespace)
 		for _, cfg := range storeConfigs {
-			key := cfg.GroupVersionKind.Kind + cfg.Namespace + cfg.Name
```

Comment:
Not: Kind is not needed since List already is for a single GVK

Reply:
is namespace needed?

Reply:
Sure, List() can list all namespace configs